### PR TITLE
Add own collections to `/collections`

### LINKED ISSUE #768: Cannot access one's own collection that's not published yet
I noticed that there's no easy way to view one's own collection that's not published yet.

For example, I can create a collaborative collection, and I can see the admin page:

image

Now, if I don't click "Make public" there, and navigate out from the page, there's no easy way to go back to the admin panel of that collection, aside from accessing the URL. (As far as I can tell.) 

It would be nice to have them listed under (1) the Collections tab or (b) the Drafts tab.

### PR DESCRIPTION
**Pull request by chartgerink**
This PR adds the collections that the workspace is an editor of to the `/collections` page - this means that it includes both public and non-public ones.

Fixes #768.

Note: This is not the best solution in the world. We can always further improve this in the future. 😊 I did not pick up the suggestion there because it would take considerably more work (I expect) to rework the `/drafts` as suggested.

## Screenshots

Screenshot 2022-10-27 at 15 30 02

### CODE REVIEW

File: app/collections/queries/getDraftCollections.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { resolver, Ctx, AuthorizationError } from "blitz"
+import db from "db"
+
+export default resolver.pipe(async ({ session }) => {
+  const draftCollections = await db.collection.findMany({
```
Comment (nsunami) [inline app/collections/queries/getDraftCollections.ts:5]: Are we getting all of one's collection, not only the draft collections, in this query? (If so, shall we change to something like `getMyCollections`?)

File: app/pages/collections.tsx [hunk 2]
```diff
@@ -38,6 +38,86 @@ const CollectionsPage: BlitzPage = () => {
...
+                                : collection.submissions.length
+                            }-teal`
+                          : `col-span-1 bg-amber-200 text-base dark:bg-amber-600 md:col-span-1 collection-${
+                              collection.submissions.length > 4
+                                ? "4"
+                                : collection.submissions.length
+                            }-amber`
+                      }  cursor-pointer rounded-md bg-cover bg-center`}
+                      key={`${collection.title}-${index}`}
+                    >
```
Comment (nsunami) [inline app/pages/collections.tsx:69]: I noticed that some of these are repeated later (under all collections), which makes a good case for refactoring. No need to resolve this on this PR—I just wanted to leave a comment 👍

**Review by nsunami — CHANGES_REQUESTED** [review]
I think this is a great patch!  (We don't want users to think that their paid collection is gone!)
Working well on my local 👍 

I had one point to clarify (the name of the query) and one comment here.

Hope that helps!

File: app/collections/queries/getDraftCollections.ts (hunk 1 above)
Comment (chartgerink) [reply app/collections/queries/getDraftCollections.ts:5]: Fair point - will update!

File: app/pages/collections.tsx (hunk 2 above)
Comment (chartgerink) [reply app/pages/collections.tsx:69]: 💯%

sloppy of me 😅 will look into refactoring this when I update the filiename

Comment (nsunami) [reply app/pages/collections.tsx:69]: No worries! And, if you want me to take over refactoring, I'm happy to do so in another issue–PR 👍

**Pull request merged**